### PR TITLE
Slapping someone with harm intent slaps them really hard

### PR DESCRIPTION
**Pull request by ynot01**
# Document the changes in your pull request

Does 2 damage

Plays louder

Has text difference

Deletes the slapper because ow my hand

image

# Changelog

:cl:  
tweak: You can now slap players really hard with harm intent
/:cl:

### CODE REVIEW

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -740,25 +740,34 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
...
-	var/slap_volume = 50
+	
+	var/slap_volume = 30
+	var/hard_slap = FALSE
+	if(!HAS_TRAIT(user, TRAIT_PACIFISM) && user.a_intent == INTENT_HARM)
+		hard_slap = TRUE
+		slap_volume = 60
+		force = 2
+		..()
+		qdel(src) // hurts your hand to slap so hard
```
Comment (Moltijoe) [inline code/game/objects/items/weaponry.dm:757]: are you utterly insane!?
it should do 3 damage

**Comment by MajManatee** [comment]
gonna slap an antag to death

**Comment by Moltijoe** [comment]
probably should close this meme PR before bibby merges it as a joke

**Comment by Moltijoe** [comment]
ah, yeah, i'm dumb, that's just deleting the slapper, not the holder

**Review by Bibby0110 — APPROVED** [review]

**Comment by ynot01** [comment]
clowns

**Pull request merged**